docs/standard/asynchronous-programming-patterns/asynchronous-programming-model-apm.md

```diff
@@ -15,31 +15,31 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # Asynchronous Programming Model (APM)
-An asynchronous operation that uses the <xref:System.IAsyncResult> design pattern is implemented as two methods named **Begin***OperationName* and **End***OperationName* that begin and end the asynchronous operation *OperationName* respectively. For example, the <xref:System.IO.FileStream> class provides the <xref:System.IO.FileStream.BeginRead%2A> and <xref:System.IO.FileStream.EndRead%2A> methods to asynchronously read bytes from a file. These methods implement the asynchronous version of the <xref:System.IO.FileStream.Read%2A> method.  
+An asynchronous operation that uses the <xref:System.IAsyncResult> design pattern is implemented as two methods named **Begin**_OperationName_ and **End**_OperationName_ that begin and end the asynchronous operation *OperationName* respectively. For example, the <xref:System.IO.FileStream> class provides the <xref:System.IO.FileStream.BeginRead%2A> and <xref:System.IO.FileStream.EndRead%2A> methods to asynchronously read bytes from a file. These methods implement the asynchronous version of the <xref:System.IO.FileStream.Read%2A> method.  
   
 > [!NOTE]
 >  Starting with the .NET Framework 4, the Task Parallel Library provides a new model for asynchronous and parallel programming. For more information, see [Task Parallel Library (TPL)](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md) and [Task-based Asynchronous Pattern (TAP)](../../../docs/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap.md)).  
   
- After calling **Begin***OperationName*, an application can continue executing instructions on the calling thread while the asynchronous operation takes place on a different thread. For each call to **Begin***OperationName*, the application should also call **End***OperationName* to get the results of the operation.  
+ After calling **Begin**_OperationName_, an application can continue executing instructions on the calling thread while the asynchronous operation takes place on a different thread. For each call to **Begin**_OperationName_, the application should also call **End**_OperationName_ to get the results of the operation.  
   
 ## Beginning an Asynchronous Operation  
- The **Begin***OperationName* method begins asynchronous operation *OperationName* and returns an object that implements the <xref:System.IAsyncResult> interface. <xref:System.IAsyncResult> objects store information about an asynchronous operation. The following table shows information about an asynchronous operation.  
+ The **Begin**_OperationName_ method begins asynchronous operation *OperationName* and returns an object that implements the <xref:System.IAsyncResult> interface. <xref:System.IAsyncResult> objects store information about an asynchronous operation. The following table shows information about an asynchronous operation.  
```

Comment:
same comment as earlier issue.

Reply:
At this point, the remaining comments are all related to the **Begin**_OperationName_ and **End**_OperationName_ formatting.  They should all be replaced with `BeginOperationName` and `EndOperationName`